lux/_config/config.py

```diff
@@ -377,11 +382,23 @@ def set_SQL_connection(self, connection):
         self.set_executor_type("SQL")
         self.SQLconnection = connection
 
+    def read_query_template(self, query_file):
+        from lux.executor.SQLExecutor import SQLExecutor
+
+        query_dict = {}
+        with open(query_file) as f:
+            for line in f:
+                (key, val) = line.split(":")
+                query_dict[key] = val.strip()
+        self.query_templates = query_dict
+        self.executor = SQLExecutor()
+
     def set_executor_type(self, exe):
         if exe == "SQL":
             from lux.executor.SQLExecutor import SQLExecutor
 
             self.executor = SQLExecutor()
+            self.read_query_template("../lux/lux/_config/postgres_query_template.txt")
```

Comment:
This seems to be an absolute path. When I tried it out the template file could not be found.
![Screen Shot 2021-06-23 at 5 00 40 PM](https://user-images.githubusercontent.com/5554675/123182546-474e7480-d4c2-11eb-87da-3ecc19ae0822.png)